### DISCUSSION TITLE
Need to host SSR files on server, instead of hosting via node server

### DISCUSSION THREAD

**Discussion by jatinlz** [getcandy/hub#66] [Help] [answered]
Hello,

i am unable to host dist folder on remote server as it doesn't have any index.html
as of now i just host same code which we have on git. using "npm run generate" and point my domain to dist/ folder.

bellow is the screenshort of dist folder which generated after above command

![Screenshot from 2021-11-16 10-52-23](https://user-images.githubusercontent.com/61220282/141925805-3ff6bc6e-2e41-4df6-ab67-d83654bafb31.png)

I'm stuck here, could you please help me ?

Thank you

**Comment by alecritson** [comment, marked as answer]
Hi @jatinlz 

The hub doesn't support SSR so you would need to use a service such as vercel/netlify or your own node server to host :)
